### PR TITLE
🧹 Remove no longer used FlowRun.timeout_on and two unused indexes

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

File: temba/flows/migrations/0242_drop_run_timeout.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Generated by Django 2.2.10 on 2020-10-29 17:18
+
+from django.db import migrations
+
+SQL = """
+DROP INDEX flows_flowrun_null_expired_on;
+DROP INDEX flows_flowrun_timeout_active;
```
Comment (rowanseymour) [inline temba/flows/migrations/0242_drop_run_timeout.py:7]: dropping the field drops the index but doing this explicitly let's collect_sql know it's been dropped

File: temba/sql/current_indexes.sql [hunk 2]
```diff
@@ -59,8 +59,6 @@ CREATE INDEX flows_flowrun_flow_modified_id ON flows_flowrun (flow_id, modified_
 
 CREATE INDEX flows_flowrun_flow_modified_id_where_responded ON flows_flowrun (flow_id, modified_on DESC, id DESC) WHERE responded = TRUE;
 
-CREATE INDEX flows_flowrun_null_expired_on ON flows_flowrun (exited_on) WHERE exited_on IS NULL;
```
Comment (rowanseymour) [inline temba/sql/current_indexes.sql:62]: Noticed from looking at index usage that this is never used (it's 1GB on RP) and I can't even figure out what it was used for...

File: temba/flows/models.py [hunk 3]
```diff
@@ -1147,9 +1147,6 @@ class FlowRun(RequireUpdateFieldsMixin, models.Model):
     # when this run will expire
     expires_on = models.DateTimeField(null=True)
 
-    # next wait timeout in this run (if any)
-    timeout_on = models.DateTimeField(null=True)
```
Comment (rowanseymour) [inline temba/flows/models.py:1151]: mailroom uses `FlowSession.timeout_on` - there's one place where clears it but that can be cleaned up https://github.com/nyaruka/mailroom/pull/379

File: temba/sql/current_indexes.sql (hunk 2 above)
Comment (norkans7) [reply temba/sql/current_indexes.sql:62]: that is probably before exited_on

**Review by norkans7 — APPROVED** [review]

File: temba/sql/current_indexes.sql (hunk 2 above)
Comment (rowanseymour) [reply temba/sql/current_indexes.sql:62]: true.. guess we renamed expired_on to exited_on at some point... hmmm

**Review by nicpottier — APPROVED** [review]

**Pull request merged**